### ISSUE TITLE
Instructions for local development

### ISSUE THREAD

**Issue by rishighan** [bpepple/metron#96]
**Is your feature request related to a problem? Please describe.**
No it isn't.

**Describe the solution you'd like**
 I want to get specific directions for spinning up Metron API locally for development reasons. I was able to:

1. Clone the repo
2. Install `python3`, `pip3`, `virtualenv` which are all pre-requisites for running Metron
3. Run `pipenv install` to install dependencies
4. Run `pip install psycopg2-binary` to install `psycopg2` on `macOS`. `#3` does not install the package and throws an error.
4. Run `python manage.py runserver` to run the `HTTP` server?

I am stuck at 4, since there are many environment variables that need to be set, and I don't know which file to edit to add them.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by bpepple** [comment]
I've started writing some basic installation instructions (warning, it's extremely Linux-centric since I haven't used Windows or a Mac since the late 90'). I still need to add a sample .env file to the repo, and double-check that I haven't missed any necessary information, but for now it will give you an idea of what needs to be done.

On a side note, I've got a family member that had a medical emergency this weekend, so my free time is most likely going to be extremely limited for the next month or so. 

Anyway, here's a link to what I've got so far: [wiki](https://github.com/bpepple/metron/wiki/Development-Installation)
